game/static/game/css/auth.css

```
@@ -19,12 +19,12 @@ body {
     min-height: 100vh;
 
     display: flex;
-    justify-content: centre;
-    gap: 10px;
+    justify-content: center;
     align-items: center;
-
     flex-direction: column;
 
+    gap: 16px;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)